src/manage-apps/manage-repos-repoman.md

```diff
@@ -0,0 +1,92 @@
+# Managing Repositories with Repoman
+
+Add, remove, and edit repositories using Repoman. Repoman allows users to add Debian repositories, including Personal Package Archives (PPAs), and Flatpak repositories. Users may want to add a Debian repository or additional Flatpak repository if their software is not included in the Pop!\_Shop, or they may want to remove PPA’s after uninstalling a package.
+
+Access Repoman by launching the Pop!\_Shop, clicking the three lines, and then clicking `System Software Sources...`.
+
+![Access Repoman](/images/manage-repos/access-repoman.png)
+
+>**Caution**: Adding 3rd-party Debian or Flatpak repositories allows installation of software that has not been validated by System76 or other trusted Linux communities. Software available via 3rd-party Debian and Flatpak repositories may not be vetted against packages that introduce security vulnerabilities. Users should take caution and only add external repositories that they trust.
+---
+
+## Official Sources
+
+This section lists System76's official Pop!\_OS mirror. This mirror includes installation files and updates for all applications included in a default Pop!\_OS installation. This is also where installation files for many applications in the Pop!\_Shop reside. Changing this mirror is not necessary since Pop!\_OS will automatically download updates from the fastest source using System76's worldwide CDN.
+
+The toggle switches enable or disable software components based on licensing and support:
+
+- **Restricted** - Proprietary drivers and components that are not open source
+- **Universe** - Free and open-source software that is community maintained
+- **Multiverse** - Software restricted by copyright or legal reasons
+
+![Official Sources Tab](/images/manage-repos/official-sources-tab.png)
+
+## Managing Debian Repositories with Repoman
+
+Debian archives (including PPAs), are created by developers to distribute software not included in default Debian repositories.
+
+### Viewing Extra Sources
+
+Click on the `Extra Sources` tab to display repositories used by 3rd-party applications.
+
+![Extra Sources](/images/manage-repos/extra-sources.png)
+
+>**Note:** While this page is generally used to add sources for 3rd-party Debian repositories, sources with a URI containing `http://apt.pop-os.org` are critical to Pop!\_OS's desktop functionality and should not be removed.
```

Comment:
I really like this note. Technically, the proprietary repo isn't critical and should be able to be disabled/removed if a user wants to minimize proprietary software available to their system. (That plus maintenance reasons on our end are the reasons it's separated out.) However, that is a big corner case, and we also host some proprietary software in the main release repo, so it's up to you if this should be clarified or not.

Reply:
I've added a description of each of these URL's and what they do in the section intro page.